WalletWasabi/WabiSabi/WabiSabiCoordinator.cs

```diff
@@ -45,20 +45,22 @@ public WabiSabiCoordinator(CoordinatorParameters parameters, IRPCClient rpc)
 
 		protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 		{
-			await ConfigWatcher.StartAsync(stoppingToken).ConfigureAwait(false);
-			await Warden.StartAsync(stoppingToken).ConfigureAwait(false);
-			await Arena.StartAsync(stoppingToken).ConfigureAwait(false);
+			await Task.WhenAll(
```

Comment:
Are we 100% sure that order does not matter?

Reply:
I'm 90% sure

Reply:
Reverted